src/main/resources/application.yml

```diff
@@ -9,6 +9,56 @@ spring:
   threads:
     virtual:
       enabled: "\${VIRTUAL_THREADS_ENABLED:true}"
+  cloud:
+    function:
+      definition: registryProducer
+    stream:
+      binders:
+        kafka-registry-producer:
+          type: kafka
+          environment:
+            spring.cloud.stream.kafka.binder:
+              brokers: "\${KAFKA_BINDER_BROKER:}"
+              configuration:
+                sasl.jaas.config: "\${KAFKA_REGISTRY_PRODUCER_SASL_JAAS_CONFIG:}"
```

Comment:
you have to open a PR on aks-deployment to set this variable, but, to do that, I have to work P4ADEV-2980 as first